### PR TITLE
LibWeb+LibJS: Move some includes around to make CSS/Parser faster

### PR DESCRIPTION
**Pull request by alimpfard**
Makes CSS/Parser.cpp about 30% less painful to edit :P

### CODE REVIEW

**Review by linusg — APPROVED** [review]
First commit also moves code around, would be nice to also mention that if you can make it fit into the title. Otherwise LGTM

**Comment by linusg** [comment]
```
Files with bad licenses: Userland/Libraries/LibJS/Heap/Cell.cpp Userland/Libraries/LibWeb/CSS/Parser/Helpers.cpp
```

not sure why tbh...

**Review by linusg — APPROVED** [review]

File: Userland/Libraries/LibWeb/CSS/Parser/Parser.cpp [hunk 1]
```diff
@@ -8746,6 +8746,59 @@ class UnparsedCalculationNode final : public CalculationNode {
     ComponentValue m_component_value;
 };
 
+// https://html.spec.whatwg.org/multipage/images.html#parse-a-srcset-attribute
```
Comment (AtkinsSJ) [inline Userland/Libraries/LibWeb/CSS/Parser/Parser.cpp:8749]: I was wondering what srcset had to do with CSS. Turns out this is just the wrong spec link, should be <https://html.spec.whatwg.org/multipage/images.html#parsing-a-sizes-attribute>. Bad Kling! No cookie!

**Comment by AtkinsSJ** [comment]
Changes look fine. But CI disagrees quite strongly!

File: Userland/Libraries/LibWeb/CMakeLists.txt [hunk 2]
```diff
@@ -53,6 +53,7 @@ set(SOURCES
     CSS/Parser/Declaration.cpp
```
Comment (LucasChollet) [inline Userland/Libraries/LibWeb/CMakeLists.txt:53]: There is a typo in the commit title :catdogalert:

Comment (alimpfard) [reply Userland/Libraries/LibWeb/CMakeLists.txt:53]: I stared at the commits for a good 20 seconds...

Comment (alimpfard) [reply Userland/Libraries/LibWeb/CMakeLists.txt:53]: :facepalm:

**Pull request merged**